regression/rt_gk_rad_1x2v_p1.c

```diff
@@ -391,7 +391,11 @@ main(int argc, char **argv)
       .num_cross_collisions = 1,
       .collide_with = { "ion" },
     },
-
+    .source = {
```

Comment:
@jRoeltgen @akashukla 
why is this needed?

Reply:
The reason for the empty source in the regression test is there was a valgrind error without it. Though I guess this doesn't actually fix the problem, just avoid it. @JunoRavin

Reply:
Oh I bet this is an issue that the source struct isn't properly initialized. I'll check.